packages/utilities/src/initials.ts

```diff
@@ -5,6 +5,12 @@
  */
 const UNWANTED_CHARS_REGEX: RegExp = /\([^)]*\)|[\0-\u001F\!-/:-@\[-`\{-\u00BF\u0250-\u036F\uD800-\uFFFF]/g;
 
+/**
+ * Regular expression matching phone numbers. Applied after chars matching UNWANTED_CHARS_REGEX have been removed
+ * and number has been trimmed for whitespaces
+ */
+const PHONENUMBER_REGEX: RegExp = /^\d+[\d\s]*(:?extension|ext|x|)\s*\d+$/i;
```

Comment:
Does "extension" need localization support?

Reply:
I've only seen extensions being used in US phone numbers. In any case, let's not include "extension", just "ext" and "x", as there are probably almost no-one who would add a contact with just a phone number and still write out "extension" in full. They would probably just use "ext" or "x".